content/id/docs/tasks/configure-pod-container/pull-image-private-registry.md

```diff
@@ -0,0 +1,212 @@
+
+## Membuat Secret berdasarkan data pribadi Docker yang sudah ada {#register-secret-data-pribadi-yang-ada}
+
+Klaster Kubernetes menggunakan Secret dari tipe `docker-registry` untuk melakukan autentikasi dengan
+register Container untuk menarik _image_ pribadi.
+
+Jika kamu sudah menjalankan `docker login`, kamu dapat menyalin data pribadi itu ke Kubernetes:
+
+```shell
+kubectl create secret generic regcred \
+    --from-file=.dockerconfigjson=<path/to/.docker/config.json> \
+    --type=kubernetes.io/dockerconfigjson
+```
+
+Jika kamu memerlukan lebih banyak kontrol (misalnya, untuk mengatur _namespace_ atau label baru pada Secret) 
+maka kamu dapat menyesuaikan Secret tersebut sebelum menyimpannya.
+Pastikan untuk:
+
+
+- Mengatur nama dari pokok (_item_) data menjadi `.dockerconfigjson`
+- Melakukan enkode secara _base64_ dari berkas Docker (_Docker file_) dan memindahkan urutan huruf (_string_) tersebut, secara tidak terputus sebagai nilai untuk bidang `data[".dockerconfigjson"]`
+- Mengatur `type` menjadi `kubernetes.io/dockerconfigjson`
+
+Sebagai contoh:
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: myregistrykey
+  namespace: awesomeapps
+data:
+  .dockerconfigjson: UmVhbGx5IHJlYWxseSByZWVlZWVlZWVlZWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWxsbGxsbGxsbGxsbGxsbGxsbGxsbGxsbGxsbGxsbGx5eXl5eXl5eXl5eXl5eXl5eXl5eSBsbGxsbGxsbGxsbGxsbG9vb29vb29vb29vb29vb29vb29vb29vb29vb25ubm5ubm5ubm5ubm5ubm5ubm5ubm5ubmdnZ2dnZ2dnZ2dnZ2dnZ2dnZ2cgYXV0aCBrZXlzCg==
+type: kubernetes.io/dockerconfigjson
+```
+
+Jika kamu mendapat pesan kesalahan `error: no objects passed to create`, ini berarti pengkodean _base64_ dari urutan huruf tersebut tidak valid.
+Jika kamu mendapat pesan kesalahan seperti `Secret "myregistrykey" is invalid: data[.dockerconfigjson]: invalid value ...`, ini berarti
+enkode _base64_ dari urutan huruf dalam data tersebut sukses didekodekan, tetapi tidak bisa di uraikan menjadi berkas `.docker/config.json`.
+
+## Membuat Secret dengan memberikan data pribadi pada baris perintah
+
+Buatlah Secret ini, dan berilah nama `regcred`:
+
+```shell
+kubectl create secret docker-registry regcred --docker-server=<your-registry-server> --docker-username=<your-name> --docker-password=<your-pword> --docker-email=<your-email>
+```
+
+dimana:
+
+* `<your-registry-server>` merupakan FQDN dari register private Docker kamu. (https://index.docker.io/v1/ untuk DockerHub)
+* `<your-name>` adalah nama pengguna Docker kamu.
+* `<your-pword>` adalah kata kunci Docker kamu.
+* `<your-email>` adalah alamat email Docker kamu.
+
+Kamu telah berhasil mengatur data pribadi untuk Docker kamu pada klaster sebagai sebuah Secret yang dipanggil dengan nama `regcred`.
+
+{{< note >}}
+
+Mengetik Secret pada baris perintah dapat menyimpannya dalam riwayat (_history_) dari _shell_ kamu tanpa perlindungan, dan
+Secret tersebut mungkin juga terlihat oleh pengguna lain dalam PC kamu selama perintah `kubectl` sedang berjalan.
+{{< /note >}}
+
+
+## Menginspeksi Secret `regcred`
```

Comment:
we can skip formatting on header now or add anchor helper
```suggestion
## Menginspeksi Secret `regcred` {#menginspeksi-secret-regcred}
```

Reply:
Agree! Fixed in https://github.com/kubernetes/website/pull/21712/commits/d8c12dd53a0aa2cd007dfc61bbeb2194a5e83cb2